model/task/db.go

```diff
@@ -2962,3 +2962,46 @@ func getGenerateTasksEstimation(ctx context.Context, project, buildVariant, disp
 
 	return results, nil
 }
+
+// getLatestTask retrieves the latest task from all the distros corresponding to the imageID
+func getLatestTask(ctx context.Context, imageID string) (*Task, error) {
```

Comment:
wondering if we should rename this function, since `getLatestTask` sounds like it's just returning the most recent task in general haha